release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -1957,7 +1957,9 @@ There is no workaround for this issue. (link:https://issues.redhat.com/browse/OC
 
 * {product-title} clusters that are installed on {aws-short} in the Mexico Central region (`mx-central-1`) cannot be destroyed. (link:https://issues.redhat.com/browse/OCPBUGS-56020[*OCPBUGS-56020*])
 
-* When installing a cluster on {azure-short}, if you set `compute.platform.azure.identity.type`, `controlplane.platform.azure.identity.type`, or `platform.azure.defaultMachinePlatform.identity.type` to "None", your cluster is unable to pull images from the Azure Container Registry. You can avoid this issue by providing a user-assigned identity, or by leaving the identity field blank, which causes the installation program to generate a user-assigned identity. (link:https://issues.redhat.com/browse/OCPBUGS-56008[*https://issues.redhat.com/browse/OCPBUGS-56008*])
+* When installing a cluster on {azure-short}, if you set `compute.platform.azure.identity.type`, `controlplane.platform.azure.identity.type`, or `platform.azure.defaultMachinePlatform.identity.type` to "None", your cluster is unable to pull images from the Azure Container Registry. You can avoid this issue by providing a user-assigned identity, or by leaving the identity field blank, which causes the installation program to generate a user-assigned identity. (link:https://issues.redhat.com/browse/OCPBUGS-56008[*OCPBUGS-56008*])
+
+* When installing a cluster on {aws-short}, if you did not configure {aws-short} credentials before running `openshift-install create install-config` or `openshift-install create cluster`, the installation program will fail. (link:https://issues.redhat.com/browse/OCPBUGS-56658[*OCPBUGS-56658*])
```

Comment:
All `create` sub-commands will hit such error, how about: _before running `openshift-install create`, the installation program will fail._

```
Available Commands:
  cluster                     Create an OpenShift cluster
  ignition-configs            Generates the Ignition Config asset
  install-config              Generates the Install Config asset
  manifests                   Generates the Kubernetes manifests
  single-node-ignition-config Generates the bootstrap-in-place-for-live-iso Ignition Config asset
```

Reply:
That sounds more accurate, I've updated the PR. Thanks